monai/metrics/wrapper.py

```diff
@@ -0,0 +1,290 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+import torch
+
+from monai.metrics.utils import do_metric_reduction, ignore_background, is_binary_tensor
+from monai.utils import MetricReduction, convert_to_numpy, convert_to_tensor, optional_import
+
+from .metric import CumulativeIterationMetric
+
+BinaryPairwiseMeasures, _ = optional_import("MetricsReloaded.metrics.pairwise_measures", name="BinaryPairwiseMeasures")
+MultiClassPairwiseMeasures, _ = optional_import(
+    "MetricsReloaded.metrics.pairwise_measures", name="MultiClassPairwiseMeasures"
+)
+
+__all__ = ["MetricsReloadedBinary", "MetricsReloadedCategorical"]
+
+
+class MetricsReloadedMetric(CumulativeIterationMetric):
+    """Base class for defining MetricsReloaded metrics as a
+    CumulativeIterationMetric in MONAI"""
+
+    def __init__(
+        self,
+        metric_name: str,
```

Comment:
Is there a way to search all the supported metric names? Better to put a link of metrics list or some Enum?

Thanks.

Reply:
How about adding two enums to `wrapper.py` with the binary and categorical metrics from MetricsReloaded that MONAI supports? Or should they be in `enums.py` maybe?

Reply:
I would suggest to put the enums in the `MetricsReloaded` project and put a link here, because I suspect we may frequently add new metrics in `MetricsReloaded`, need to maintain the list out of MONAI?

Thanks.

Reply:
Okay, that sounds like a better idea, let's do that.